src/pages/community-champions.js

```diff
@@ -353,6 +359,90 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    color: ${({ theme }) => theme.colorMap.greyDarkOne};
+    margin-bottom: ${LINE_BOTTOM_MARGIN};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const GreenBulletedListWithGreyText = styled(GreenBulletedList)`
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+    p {
+        color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    }
+`;
+
+const OtherDetailsAndRequirementsContainer = styled('div')`
+    display: grid;
+    grid-template-columns: ${OTHER_DETAILS_AND_REQUIREMENTS_IMAGE_SIZE} auto;
+    @media ${screenSize.upToMedium} {
+        ul {
+            /* Show list under image instead */
+            display: none;
+        }
+    }
+`;
+
+const OtherDetailsAndRequirementsTextSection = styled('div')`
+    h5 {
+        @media ${screenSize.upToMedium} {
+            margin-bottom: ${size.tiny};
+        }
+    }
+    p {
+        max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_DESCRIPTION_MAX_WIDTH};
+    }
+    ul {
+        max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_LIST_MAX_WIDTH};
+    }
+`;
+
+const OtherDetailsAndRequirementsImageContainer = styled('div')`
+    display: flex;
+    align-items: start;
+    justify-content: center;
+`;
+
+const OtherDetailsAndRequirementsImage = styled('img')`
+    width: ${OTHER_DETAILS_AND_REQUIREMENTS_IMAGE_WIDTH};
+`;
+
+const OtherDetailsAndRequirementsBullets = () => (
+    <GreenBulletedListWithGreyText
+        children={[
+            'MongoDB Champions must (re)qualify every year, based on their contribution to the community',
+            'Champions program cohorts are intentionally limited to small, functional groups',
+            'Commitment of at least one full year',
+        ]}
+    />
+);
+
+const OtherDetailsAndRequirements = ({ isMobile = false }) => (
+    <div>
+        <OtherDetailsAndRequirementsContainer>
+            <OtherDetailsAndRequirementsImageContainer>
+                <OtherDetailsAndRequirementsImage
+                    src={PaperAndPencilImage}
+                    alt="Paper with a diagram drawn on it with a pencil next to it"
+                />
+            </OtherDetailsAndRequirementsImageContainer>
+            <OtherDetailsAndRequirementsTextSection>
+                <H5>Other Details &amp; Requirements</H5>
+                <Description collapse>
+                    We expect members to participate actively, collaborate
+                    closely, and work towards tangible outcomes together.
+                </Description>
+                <OtherDetailsAndRequirementsBullets />
+            </OtherDetailsAndRequirementsTextSection>
+        </OtherDetailsAndRequirementsContainer>
+        {isMobile && <OtherDetailsAndRequirementsBullets />}
```

Comment:
Would a css media query with `display: none` do the same thing here as trying to pass isMobile around? Ideally we wouldn't need to use `useMobile` for the flashing reason

Reply:
great suggestion! im sure a media query would work. i'm going to try using `grid-template-areas` first. this would avoid us having to use 2 nested containers...

Reply:
Okay sweet that _should_ help reduce the complexity here